python_ta/checkers/constant_test_checker.py

```diff
@@ -0,0 +1,55 @@
+"""
+checker for using constant value in a conditional statement
+"""
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+
+
+class UsingConstantTestChecker(BaseChecker):
+
+    __implements__ = IAstroidChecker
+
+    name = 'using-constants-test'
+
+    msgs = {'W7777': ('using a constant value in a conditional statement'
+                      , 'using-constants-test'
+                      , 'Conditional statements should depend on a variable not a constant value.'
```

Comment:
Change text to: `'This condition is a constant expression, meaning the same branch will always be executed (and the other branch does not have a purpose).'`